src/nifs/protogalaxy/accumulator.rs

```diff
@@ -63,6 +67,17 @@ pub struct AccumulatorInstance<C: CurveAffine> {
     pub(super) e: C::ScalarExt,
 }
 
+impl<C: CurveAffine> AccumulatorInstance<C> {
+    pub fn into_acc(self, w: PlonkWitness<C::Scalar>) -> Accumulator<C> {
```

Comment:
maybe a different name?  `into_acc` seems like you can directly convert Instance into Accumulator without extra data.

Reply:
Well here the function requires a parameter, so it's hardly confusing to anyone. I used the standard Rust naming convention, the alternative is to make an `Accumulator` constructor, something like `from(PlonkInstance, PlonkWitness)`. Let me know if you find this option preferable

Reply:
I think it's fine, didn't think of a better name. both "into" or "from" are equivalent naming.